apps/ui/components/search/SingleSearchReservationUnitCard.tsx

```diff
@@ -289,6 +301,7 @@ function ReservationUnitCard({ reservationUnit }: PropsT): JSX.Element {
               closed: reservationUnit.isClosed ?? false,
               availableAt: reservationUnit.firstReservableDatetime ?? "",
             }}
+            id={reservationUnit.pk}
```

Comment:
Well, `id="#pk"` is unique, which was the only criteria here (referring to the discussion when I asked about  whether "diipadaapa-id":s would suffice, as long as they're unique [answer: yes]).
But I completely agree: simply `id="1"` would be less informative (and has more chances to possibly clash with some other `id`) than `a-relevant-prefix-1`. Not to mention unnecessary, since it's trivial to add some prefix to it.
That's why there is a prefix (`status-tag-`) added in the `StatusTag` component, to whatever is declared here. So we would never have a `Tag` with `id="1"`, as using "1" here would result in "status-tag-1". Thus I don't really think this comment is valid.

However I'll move the prefix addition from `StatusTag` to be done here, as it's clearer and also the simplest way to resolve the other comments. That way the `id` is directly used as the resulting html `id` (making the prop name less awful, and making declaring a const for the id string in `StatusTag` unnecessary).